SingularityUI/webpack.config.js

```diff
@@ -52,7 +52,8 @@ module.exports = {
       'handlebars': 'handlebars/runtime.js',
       'sortable': 'sortable/js/sortable.js',
       'datatables': 'datatables/media/js/jquery.dataTables.js',
-      'bootstrap': 'bootstrap/dist/js/bootstrap.js'
+      'bootstrap': 'bootstrap/dist/js/bootstrap.js',
+      'react': path.join(__dirname, 'node_modules', 'react')
```

Comment:
Good question,

I needed to do some development on react-typeahead, and I was using `npm link` to do so.
This line makes sure that there is only one version of react being used at runtime, because react-typeahead required react locally to build.

I left it in because I anticipated needing the fix to be in place in the future. YMMV.

I can take it out if you want.